src/main/java/pl/allegro/tech/boot/autoconfigure/handlebars/HandlebarsHelpersAutoConfiguration.java

```diff
@@ -117,6 +118,19 @@ public void registerHelpers() {
         }
     }
 
+    @Configuration
+    @ConditionalOnClass(StringHelpers.class)
+    static class StringHelpersAutoConfiguration {
+
+        @Autowired
+        private HandlebarsViewResolver handlebarsViewResolver;
+
+        @PostConstruct
+        public void registerHelpers() {
+            StringHelpers.register(handlebarsViewResolver.getHandlebars());
```

Comment:
Do we have to register it on our own? It is a [built-in helper](https://github.com/jknack/handlebars.java#built-in-helpers).